src/webui/www/private/views/rssDownloader.html

```diff
@@ -478,6 +503,13 @@
             });
         };
 
+        const removeSelectedRule = () => {
+            if (rssDownloaderRulesTable.selectedRows.length > 0) {
+                removeRule(rssDownloaderRulesTable.selectedRows
+                    .map((sRow) => rssDownloaderRulesTable.rows[sRow].full_data.name));
```

Comment:
You mean the return value of the map? We are using it since removeRule takes in an array of rules.
So this doesn't work:
```js
rssDownloaderRulesTable.selectedRows.forEach((sRow) => 
    removeRule(rssDownloaderRulesTable.rows[sRow].full_data.name));
```
<br />
I renamed the function to better reflect that.

Reply:
I read it incorrectly earlier, but I see it now.